src/components/ResultsItemDetails.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <section>
+    <article class='article-button'>
+      <button @click="$emit('save:park', parkName)" class='button-save-park'>SAVE</button>
+    </article>
+    <article class='article-destination'>
+      <h1 class='detail-descriptor'>Destination: </h1>
+      <p>Address: </p>
+    </article>
+    <article class='article-description'>
+      <p>Open Now?</p>
+      <p>Rating: /5</p>
+      <splide :options="options">
+        <splide-slide v-for="(photo, i) in photos" :key="i" >
+          <img :src="photo.src" :alt="'photo for ' + parkName" />
+        </splide-slide>
+      </splide>
+    </article>
+    <button class='button-get-directions'>Get Directions</button>
+    <!-- button is not functional yet - need to get a directions component w/ router -->
+    <h2>Not the right park for your pup?</h2>
+    <router-link to='/'><button>search again</button></router-link>
+  </section>
+</template>
+
+<script>
+import { Splide, SplideSlide } from '@splidejs/vue-splide'
+import '@splidejs/splide/dist/css/themes/splide-sea-green.min.css';
+
+export default {
+  components: {
+    Splide,
+    SplideSlide
+  },
+  inject: ['savePark'],
```

Comment:
That's because I have both `savedParks [ ]` and `savePark( )`, only `savePark` the method is being injected.  `savedParks` isn't being used yet but figured we'd need it in `App` at some point.  Perhaps I can change the name of either so they aren't too similar.

Reply:
Oh gotcha, that makes total sense! I think the naming convention is fine, I was reviewing this first thing and the morning and missed that. Sorry about that!

Reply:
That totally makes sense! I just missed the connection with it, don't think the names are too similar at all. Sorry about that!

Reply:
No worries! Glad we got it cleared up.